docs/diagrams/UndoRedoVersion1.puml

```diff
@@ -0,0 +1,23 @@
+@startuml
+!include style.puml
+skinparam ClassFontColor #000000
+skinparam ClassBorderColor #000000
+skinparam ClassBackgroundColor #FFFFAA
+
+title After command "deleteMember 5"
+
+package Versions <<rectangle>> {
```

Comment:
What does the <<rectangle>> here do?

Reply:
I think putting `<<rectangle>>` here does not affect the diagram output. It is fine to leave it in.
.